hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -1,11 +1,73 @@
 module HookContextExtension
+  # FIXME: remove when #23332 is released
+  def param_value(mod, name)
+    param(mod, name).value if param(mod, name)
+  end
+
   def success_file
     File.join(File.dirname(kafo.config.config_file), '.installed')
   end
 
   def new_install?
     !File.exist?(success_file)
   end
+
+  def ensure_package_installed(package)
+    bin_path = Kafo::PuppetCommand.search_puppet_path('puppet')
+    execute("#{bin_path} resource package #{package} ensure=installed")
```

Comment:
Now that I read more, would it make sense to have the ensure state as a parameter? There's still a `yum -y remove` in the same code block below and `ensure=absent` would allow us to abstract that away as well. I can also imagine we may want a `ensure=latest`
```suggestion
    execute("#{bin_path} resource package #{package} ensure=#{ensure}")
```